scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferWithRawSqlProcessor.java

```diff
@@ -0,0 +1,158 @@
+package kelpie.scalardb.transfer;
+
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.storage.jdbc.JdbcDatabaseConfig;
+import com.scalar.db.storage.jdbc.JdbcUtils;
+import com.scalar.db.storage.jdbc.RdbEngine;
+import com.scalar.kelpie.config.Config;
+import com.scalar.kelpie.modules.TimeBasedProcessor;
+import kelpie.scalardb.Common;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class TransferWithRawSqlProcessor extends TimeBasedProcessor {
```

Comment:
```suggestion
public class SqlBasedNontransactionalTransferProcessor extends TimeBasedProcessor {
```
Or, `JdbcBasedXXX`?

Anyway, I know it's quite long...